### DISCUSSION TITLE
Why is the published page different from the editor and preview interface?? The UI of the published page is very ugly, not as good as the previous interface., Suggest solving it

### DISCUSSION THREAD

**Discussion by HVGNc** [GitbookIO/community#562] [0. General bugs and questions]
Why is the published page different from the editor and preview interface?? The UI of the published page is very ugly, not as good as the previous interface., Suggest solving it

As shown in the figure, the readability and UI of the published webpage interface are very poor
What is inside the scriptwriter is the previous UI, how can we restore it to the previous UI?

As shown in the figure, Figure 2 is the previous UI that looked very nice and had strong readability
![12](https://github.com/GitbookIO/community/assets/87613584/c07073a5-b474-4dcb-8505-8171efc10135)
![22](https://github.com/GitbookIO/community/assets/87613584/e045802a-c3ec-4188-a16f-f764c480bd30)

**Comment by HVGNc** [comment]
As shown in the figure, Figure 2 is the previous UI that looked very nice and had strong readability

**Comment by taszap** [comment]
Hi @HVGNc 

Thanks for your feedback. As my colleague mentioned in #554 - we are continuously shipping improvements. 

> We'll look at iterating on the readability of content, both in the editor and in published content, and align the 2 experiences.

We announce all our changes in our changelog. 
I will close this discussion in favour of #554
